gremlin-javascript/src/main/javascript/gremlin-javascript/lib/structure/io/type-serializers.ts

```diff
@@ -349,9 +349,9 @@ export class EdgeSerializer extends TypeSerializer<g.Edge> {
       [valueKey]: {
         id: this.writer.adaptObject(item.id),
         label: item.label,
-        outV: this.writer.adaptObject(item.outV.id),
+        outV: this.writer.adaptObject(item.outV?.id),
```

Comment:
Can `outV` really be `undefined`? And shouldn't users get an exception here if it is?

Reply:
Oops, nice catch, I've removed the unnecessary optional chaining.